CarRecognition/Source Files/Modules/Recognition/Root/CarCardViewController/CarCardView.swift

```diff
@@ -162,11 +177,24 @@ internal final class CarCardView: View, ViewSetupable {
             view.invalidateChart(animated: true)
         }
     }
+    
+    /// Animates attach pin error label
+    func animateAttachPinError() {
+        UIView.animate(withDuration: 0.3, delay: 1.0, options: [], animations: {
```

Comment:
Animation duration `0.3` is repeated twice. What do you think about moving this to constant?

Also showing this kind of SnackBar could be moved to some kind of struct called for example `SnackBarDisplayer` with just one public method `show(in view: UIView)`. This view is getting dangerously big.

Reply:
Adding duration to constants seems fine. In terms of this `SnackBarDisplayer` how about position of this SnackBar?

Reply:
You could set it up exactly as you are doing it right now. It doesn't need to know anything about the content of the view because it uses only anchors.

Reply:
I'm using top anchor of scan button, so I would need to know context :/

Reply:
But I can also just move it on the bottom :)